azimuth_shr/loading_resources.py

```diff
@@ -67,3 +67,32 @@ def load_CLINC150_data(full_path, python_loader, train=True) -> Dataset:
     if not train:
         dst.pop("train")
     return dst
+
+
+def align_labels(ds_dict: DatasetDict, azimuth_config: AzimuthConfig) -> DatasetDict:
+    features: Features = [v.features for v in ds_dict.values()][0]
```

Comment:
I know this is not new, but while we are here, we can grab the first value in `ds_dict` without doing the rest of the iteration, and without creating a new comprehension list:
```suggestion
    features: Features = next(iter(ds_dict.values())).features
```

Reply:
Oh, awesome hack, thanks!